Example/Showcase/Classes/PresentationLayer/Chat/Welcome/WelcomeInteractor.swift

```diff
@@ -1,9 +1,17 @@
 import Foundation
+import Combine
+
 import WalletConnectRelay
+import WalletConnectPairing
+import Auth
 
 final class WelcomeInteractor {
+    private var disposeBag = Set<AnyCancellable>()
+    
     private let chatService: ChatService
     private let accountStorage: AccountStorage
+    
+    var authClient: AuthClient?
```

Comment:
Why authClient is setting in generateUri? If important to initialize it lazily it could be lazy var. 
Didn't find, why we are store it?